src/engine/common/homestore_utils.cpp

```diff
@@ -73,5 +76,49 @@ sisl::byte_array hs_utils::extract_byte_array(const sisl::byte_view& b, const bo
     return (is_aligned_needed) ? b.extract(alignment) : b.extract(0);
 };
 
+std::string hs_utils::encodeBase64(const uint8_t* first, std::size_t size) {
+    using Base64FromBinary = boost::archive::iterators::base64_from_binary<
+        boost::archive::iterators::transform_width< const char*, // sequence of chars
+                                                    6,           // get view of 6 bit
```

Comment:
view: 6 bit
Sequence: 8 bit

Is it expected in decode function below:
View is set to 8 bit
Seq is set to 6 bit

Reply:
correct. During encoding, every sequence of 8 bits are converted to the view of 6 bits. In contrast, in decode every 6 bits are converted to 8 bits.
in base X64 encode BitsOut = 6, BitsIn = 8